app/src/ckeditor.ts

```diff
@@ -92,16 +96,20 @@ ClassicEditor.create(sourceElement, {
   plugins: [
     ...imagePlugins,
     Alignment,
+    Autoformat,
     Autosave,
     BlockQuote,
     Bold,
+    Code,
     CodeBlock,
     ContentLinks,
     ContentClipboard,
     Differencing,
     Essentials,
+    FindAndReplace,
     Heading,
     Highlight,
+    ImageTextAlternative,
```

Comment:
Did you consider putting this into the Array with image plugins?
What happens if line 97 gets removed? This plugin probably will do nothing and should be removed too?